.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
Please leave one extra line after .env
If you line 1 is the last line of source file, then line 2 should be blank.